src/js/dp/release-calendar.js

```diff
@@ -30,3 +36,118 @@ document.addEventListener("DOMContentLoaded", function () {
     releaseTypeAutoSubmit(".release-calendar__filters .filters__release-type");
   }
 });
+
+const sortSelector = document.querySelector(".ons-input--select");
+if (sortSelector) {
+  sortSelector.addEventListener("change", async (e) => {
+    gtmDataLayerPush({
+      event: "SortBy",
+      "sort-by": e.target.value,
+    });
+  });
+}
+
+if (releaseCalendarContainer) {
+  [
+    ...releaseCalendarContainer.querySelectorAll(
+      ".ons-radio__input[type=radio]:not(input:disabled)"
+    ),
+  ].map((topFilter) => {
+    topFilter.addEventListener("change", async (e) => {
+      gtmDataLayerPush({
+        event: "Filter",
+        "filter-by": e.target.name,
+        selected: e.target.value.replace("type-", ""),
+      });
+    });
+  });
+
+  document.querySelector('[aria-label="Keywords"]').onsubmit = async (e) => {
```

Comment:
I don't think this needs `async`, similar reasons as above

Reply:
We shouldn't search the DOM for an aria label to hang logic off it, because these labels will be subject to localisation as they're for human consumption. There should be a stable, machine orientated value that the logic is connected to. Classes are often used in the design system for this purpose (which you can invent as needed), for example `release-calender-analytics-js` or perhaps some other name more specific to this task.

Reply:
this is done now